k2/requirements-dev.txt

```diff
@@ -10,7 +10,6 @@ flake8-string-format==0.2.3
 flake8==3.7.6
 flake8_tuple==0.2.13
 isort==4.3.0
-pdoc==0.3.2
```

Comment:
I ran into pip problems with package and the solution seems to be to switch to pdoc3, but I also found no references to pdoc so I removed it instead.